### DISCUSSION TITLE
Which grammar format does ANTLR expect/support?

### DISCUSSION THREAD

**Discussion by dvob** [antlr/antlr4#3296] [Q&A] [answered]
Which grammar format does ANTLR expect/support? I'm very new to this whole topic of parsers so maybe this is super obvious to everyone else.
As far as I understand I can pass EBNF to ANTLR. Is that true? Does it also support other formats? 
Why isn't that mentioned somewhere? Is EBNF the de facto standard and if nothing is mentioned we can assume that EBNF is meant if we speak about context-free grammar formats?

**Comment by ericvergnaud** [comment, marked as answer]
please move this to the google discussion group

**Reply by dvob** [reply]
https://groups.google.com/g/antlr-discussion/c/YZJKGHYA_jQ
